### PR TITLE
Improved readability of code

### LINKED ISSUE #46: Improve readability of code
We need to improve readability of code 
Like instead of 
```Python
import datetime as dt
```
using `datetime` directly makes it more readable 
similarly at some places there are variables like `u`, `r` 
let's rename them to something like `url_of_something` 
for this one 
```Python 
for i in add:
```
https://github.com/ganeshhubale/readit/blob/master/manager/readit.py#L24
we can write it as 

```Python 
for url_to_add in add:
```

### PR DESCRIPTION
**Pull request by ganeshhubale**
Fix #46 
Updated normal variable names to meaningful one.

### CODE REVIEW

File: manager/readit.py [hunk 1]
```diff
@@ -18,71 +18,71 @@ def main(insert, add, tag, delete, clear, update, search, view, version):
     """
     Readit - Command-line bookmark manager tool.
     """
-    d = db.DatabaseConnection()
+    object_database = database.DatabaseConnection()
```
Comment (bhavin192) [inline manager/readit.py]: I think we should call this as `database_object` or `database_connection`

File: setup.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
             'readit = manager.__init__:main',
         ],
     },
-    version="0.1",
+    version="1.0",
```
Comment (bhavin192) [inline setup.py]: If I remember correctly, we are going to release the `v0.1`.

**Review by bhavin192 — CHANGES_REQUESTED** [review]

File: manager/readit.py (hunk 1 above)
Comment (ganeshhubale) [reply manager/readit.py]: @bhavin192 
Yes, database_connection can be better.

File: setup.py (hunk 2 above)
Comment (ganeshhubale) [reply setup.py]: @bhavin192 
yes, `v0.1`
I will update it.

**Pull request merged**